app/controllers/users_controller.rb

```diff
@@ -1,5 +1,7 @@
 class UsersController < ApplicationController
-  def new; end
+  def new
+    @user = User.new(user_params)
+  end
```

Comment:
Nice job getting it to refill if they got something wrong

Reply:
Love it